framework/src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -405,31 +405,33 @@ public static void setParam(final Config config) {
 
     if (StringUtils.isNoneBlank(PARAMETER.privateKey)) {
       localWitnesses = (new LocalWitnesses(PARAMETER.privateKey));
+      byte[] witnessAddress = null;
       if (StringUtils.isNoneBlank(PARAMETER.witnessAddress)) {
-        byte[] bytes = Commons.decodeFromBase58Check(PARAMETER.witnessAddress);
-        if (bytes != null) {
-          localWitnesses.setWitnessAccountAddress(bytes);
+        witnessAddress = Commons.decodeFromBase58Check(PARAMETER.witnessAddress);
+        if (witnessAddress != null) {
           logger.debug("Got localWitnessAccountAddress from cmd");
         } else {
           PARAMETER.witnessAddress = "";
           logger.warn(IGNORE_WRONG_WITNESS_ADDRESS_FORMAT);
         }
       }
-      localWitnesses.initWitnessAccountAddress(PARAMETER.isECKeyCryptoEngine());
+      localWitnesses.initWitnessAccountAddress(witnessAddress, PARAMETER.isECKeyCryptoEngine());
       logger.debug("Got privateKey from cmd");
     } else if (config.hasPath(Constant.LOCAL_WITNESS)) {
```

Comment:
In the following two scenarios, why not consider the `witnessAddress` configuration of the startup command, which should have the highest priority?
1. else if (config.hasPath(Constant.LOCAL_WITNESS))
2. (config.hasPath(Constant.LOCAL_WITNESS_KEYSTORE))